Wikipedia/Code/WMFArticlePreviewTableViewCell.m

```diff
@@ -30,11 +34,22 @@ @implementation WMFArticlePreviewTableViewCell
 
 #pragma mark - Setup
 
+- (void)blurAndShowLoadingIndicator {
```

Comment:
API tip: these kinds of methods which are "mirror images" of each other and change what is effectively the same state can be refactored into a single property. for example: `@property (nonatomic, assign) BOOL loading`. this will simplify the `if` statement in the cell configuration block to: `cell.loading = self.isFetching`.